src/Schema/Events/Tap.ts

```diff
@@ -327,6 +327,29 @@ export interface TappedFairGroup extends TappedEntityGroup {
   action: ActionType.tappedFairGroup
 }
 
+/**
+ * A user taps a grouped card (Explore By, Discover Something new)
```

Comment:
**suggestion**: "A user taps a grouping of cards (one of our supported home view section types)"? Maybe drop the specific "Explore By" / "Discover Something New" examples. Not sure if that's super meaningful out of context.